### PR TITLE
[ Docs - Guides] - Introduction

### PR DESCRIPTION
**Pull request by elenatorro**
Related issue: https://github.com/CartoDB/cartoframes/issues/990

### CODE REVIEW

File: docs/developer-center/guides/01-Introduction.md [hunk 1]
```diff
@@ -1 +1,39 @@
...
+
+* [Quickstart]({{ site.url }}/developers/cartoframes/guides/Quickstart/)
+
+Learn the common steps data scientists follow in their **spatial analyses** and how CARTOframes library helps in each of them by following a **real use case**.
+
+* [Install CARTOframes in your Notebooks]({{ site.url }}/developers/cartoframes/guides/Install-CARTOframes-in-your-Notebooks/)
+
+Learn how to install the CARTOframes Python library in a **Jupyter Notebook**.
+
+* [Visualize your Data - Part 1]({{ site.url }}/developers/cartoframes/guides/Visualize-your-Data-Part-1/)
```
Comment (newLadyAga) [inline docs/developer-center/guides/01-Introduction.md]: Visualize your Data - Basic

Comment (newLadyAga) [inline docs/developer-center/guides/01-Introduction.md]: [Visualize your Data - Basic]

File: docs/developer-center/guides/01-Introduction.md [hunk 2]
```diff
@@ -1 +1,39 @@
...
+
+Learn the basics to visualize data with CARTOframes: understand how to add different types of **layers**, change the **basemap**, configure the **viewport**.
+
+* [Visualize your Data - Part 2]({{ site.url }}/developers/cartoframes/guides/Visualize-your-Data-Part-2/)
```
Comment (newLadyAga) [inline docs/developer-center/guides/01-Introduction.md]: [Visualize your Data - Advanced]

File: docs/developer-center/guides/01-Introduction.md [hunk 3]
```diff
@@ -1 +1,39 @@
...
+
+Learn how to download, upload and update data with CARTOframes. Understand how to use **datasets**, **dataframes** and run **SQL** queries.
+
+* [Use Location Data Services]({{ site.url }}/developers/cartoframes/guides/Use-Location-Data-Services/)
+
+Learn to apply **geocoding** and **isolines** services to your data, and visualize the results.
+
+* [Discover Data]({{ site.url }}/developers/cartoframes/guides/Discover-Data/)
+
+Explore the and use **Data Observatory** methods to discover data in our **catalog**.
```
Comment (newLadyAga) [inline docs/developer-center/guides/01-Introduction.md]: Explore the **Data Observatory** and use its methods to discover data in our **catalog**

File: docs/developer-center/guides/01-Introduction.md [hunk 4]
```diff
@@ -1 +1,39 @@
...
+
+* [Enrich Your Datasets]({{ site.url }}/developers/cartoframes/guides/Enrich-Your-Datasets/)
+
+Understand how to enrich **points** and **polygons** with data from our **catalog**, using dataset **variables** and applying **filters**. 
+
+* [Publish and share]({{ site.url }}/developers/cartoframes/guides/Publish-and-share/)
+
+Learn how to publish your results and share them with the world.
```
Comment (newLadyAga) [inline docs/developer-center/guides/01-Introduction.md]: Learn how to publish your results and configure their privacy to share them with the world.

**Comment by Jesus89** [comment]
:)

**Pull request closed without merging**